<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuideDlg</class>
 <widget class="QWidget" name="GuideDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>初始化</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>服务器IP地址</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>服务器端口号</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>通讯密钥</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editServerAddress"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editServerKey"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxServerPort">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editServerAddress</tabstop>
  <tabstop>spinBoxServerPort</tabstop>
  <tabstop>editServerKey</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
